ner/src/main/java/edu/illinois/cs/cogcomp/ner/LbjTagger/NEDisplayPredictions.java

```diff
@@ -64,19 +67,19 @@ public static void test(String testDatapath, String dataFormat, boolean verbose)
     public static void reportPredictions(Data dataSet) {
```

Comment:
the name "reportPredictions()" and the void return type suggest STDOUT instead of logger in this method